Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Image to use 
+FROM python:3
+ADD hello_world.py /
+
+RUN pip install flask
+RUN pip install flask_restful
+EXPOSE 3333
```

Comment:
What is the purpose of these three lines - installing flask and expose a port? I think they are not needed and could be removed. Could you please remove them?